lnbits/wallets/lndgrpc.py

```diff
@@ -139,21 +139,25 @@ async def create_invoice(
         if description_hash:
             data["description_hash"] = description_hash
         elif unhashed_description:
-            data["description_hash"] = hashlib.sha256(
-                unhashed_description
-            ).digest()  # as bytes directly
+            data["description_hash"] = hashlib.sha256(unhashed_description).digest()
 
         try:
             req = ln.Invoice(**data)
             resp = await self.rpc.AddInvoice(req)
         except Exception as exc:
             logger.warning(exc)
-            error_message = str(exc)
-            return InvoiceResponse(False, None, None, error_message)
+            return InvoiceResponse(ok=False, error_message=str(exc))
 
         checking_id = bytes_to_hex(resp.r_hash)
         payment_request = str(resp.payment_request)
-        return InvoiceResponse(True, checking_id, payment_request, None)
+        # TODO: add preimage
+        preimage = None
```

Comment:
This is the report for the original issue: (https://github.com/lnbits/lnbits/issues/3078)
The preimage is is present in the response?